mason-versions.ini

```diff
@@ -1,14 +1,14 @@
 [headers]
 boost=1.66.0
 geometry=96d3505
-gzip-hpp=0.1.0
 protozero=1.6.2
 spatial-algorithms=0.1.0
 variant=1.1.5
 vtzero=2915725
 [compiled]
+libdeflate=e9d1014
```

Comment:
This can be updated to 1.0 now since @artemp tagged 1.0 in mason.

Reply:
https://github.com/mapbox/vtcomposite/commit/65906152048aea99c528af6e8329502327decf41